### PR TITLE
#3676 - Declaration content updates

### PR DESCRIPTION
**Pull request by katrina-aot-git**
Screenshot 2024-10-04 at 11 09 16 AM
Screenshot 2024-10-04 at 11 09 46 AM
Screenshot 2024-10-04 at 11 10 14 AM
Screenshot 2024-10-04 at 11 11 06 AM

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/studentprofile.json [hunk 1]
```diff
@@ -3597,7 +3597,7 @@
           "tags": [],
           "properties": {},
           "conditional": {
-            "show": null,
+            "show": "",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/studentprofile.json:3600]: On create student , the declaration is still not modified. 
image

File: sources/packages/forms/src/form-definitions/supportinguserspartner2022-2023.json [hunk 2]
```diff
@@ -4119,7 +4116,7 @@
...
+                      "html": "<p>I hereby consent to the disclosure of information from my income tax records, by the Canada Revenue Agency to an official of the Ministry of Post-Secondary Education and Future Skills, that pertains to information provided on any StudentAid BC application. The information disclosed will be relevant to, and used solely for the purpose of, determining and verifying eligibility for student aid under the Canada Student Financial Assistance Act. The information disclosed by the Canada Revenue Agency to the Ministry of Post Secondary and Future Skills will be protected from unauthorized use or disclosure and will not be disclosed to any other person or organization without my approval. This authorization is valid for the two taxation years prior to the year of signature of this consent, and the year of signature.</p>\n",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/supportinguserspartner2022-2023.json:4119]: There is one more occurrence of Post Secondary and Future skills. 

image

File: sources/packages/forms/src/form-definitions/studentprofile.json [hunk 3]
```diff
@@ -3763,14 +3763,14 @@
...
+      "content": "Collection and Use of Information. The information included in this form is collected under ss. 26(c) and 26(e) of the Freedom of Information and Protection of Privacy Act, R.S.B.C. 1996, c. 165. Upon pressing the “Submit” button you are confirming that you have reviewed this statement. The information you provide will be used in confirming your identity. If you have any questions about the collection and use of this information, contact the Executive Director, StudentAid BC, Ministry of Post-Secondary Education and Future Skills, PO Box 9173, Stn Prov Govt, Victoria BC, V8W 9H7, telephone 1-800-561-1818 (toll-free in Canada/U.S.) or +1-778-309-4621 from outside North America.",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/studentprofile.json:3773]: Unless I am missing something, this was not captured in ticket but it is good that you fixed the text.

File: sources/packages/forms/src/form-definitions/studentprofile.json (hunk 1 above)
Comment (katrina-aot-git) [reply sources/packages/forms/src/form-definitions/studentprofile.json:3600]: have updated the name across files now

File: sources/packages/forms/src/form-definitions/supportinguserspartner2022-2023.json (hunk 2 above)
Comment (katrina-aot-git) [reply sources/packages/forms/src/form-definitions/supportinguserspartner2022-2023.json]: have updated the name across files now

File: sources/packages/web/src/constants/message-constants.ts [hunk 4]
```diff
@@ -18,4 +18,5 @@ export const MINISTRY_SHORTNAME = "PSFS";
 /**
  * Ministry Fullname
  */
-export const MINISTRY_NAME = "Ministry of Post Secondary and Future Skills";
```
Comment (dheepak-aot) [inline sources/packages/web/src/constants/message-constants.ts:21]: This changes the Logo. Do we have the requirement to change the verbiage all around application? 
image

Or just about the content related to `Consent`?

Comment (andrewsignori-aot) [reply sources/packages/web/src/constants/message-constants.ts:21]: I guess that business did not anticipate it but it seems to be a desired side-effect.
I would collect some screenshots from the impacted places and add them to the ticket comments.
@JerPearson can give his approval.

Comment (katrina-aot-git) [reply sources/packages/web/src/constants/message-constants.ts:21]: Turns out the only place that that constant is currently being used is that logo. Other places that have the ministry name include the file uploader.
Screenshot 2024-10-04 at 3 34 40 PM
Screenshot 2024-10-04 at 3 36 21 PM
Screenshot 2024-10-04 at 3 41 57 PM

**Review by andrepestana-aot — COMMENTED** [review]
Looks good to me, The only is the `MINISTRY_NAME` constant that needs to be confirmed with biz.

File: sources/packages/web/src/constants/message-constants.ts (hunk 4 above)
Comment (katrina-aot-git) [reply sources/packages/web/src/constants/message-constants.ts:21]: Jeremy's away today, Josh took a look and approved the additional change. :)

**Review by dheepak-aot — APPROVED** [review]
Looks good 👍

**Review by andrepestana-aot — APPROVED** [review]
Thanks for confirming with biz.

**Pull request merged**